macro_extras/README.md

```diff
@@ -577,13 +598,9 @@ The naming is performed using the function ``unpythonic.misc.namelambda``, which
  - Expression-assignment to an unpythonic environment, ``f << (lambda ...: ...)``
 
  - Let bindings, ``let[(f, (lambda ...: ...)) in ...]``, using any let syntax supported by unpythonic (here using the haskelly let-in just as an example).
+ - Env-assignments are now processed lexically, just like regular assignments. Added support for let-bindings.
```

Comment:
Some of the wording left over from the old changed-in-version-x note, should be:

---

- Env-assignments are processed lexically, just like regular assignments.

---

(Extra "now" in the first sentence. Also, support for let-bindings is already explained above, so it's not needed here.)